WordPress/src/main/res/layout/new_account_user_fragment_screen.xml

```diff
@@ -87,6 +87,7 @@
                     android:layout_height="wrap_content"
                     style="@style/WordPress.NUXEditText"
                     android:inputType="textUri"
+                    android:digits="0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Right, I agree with you. 

Noticed that, as reported in a comment in the ticket, we've the same problem in the Blog Address field. One can use capital letters, and punctuation there. Would be good to apply the same fix.